ipaclient/install/ntpconf.py

```diff
@@ -20,6 +20,7 @@
 import os
```

Comment:
Should we call this file `timeconf.py` instead?

Reply:
I believe we should but we must not. :smile: